src/io/flutter/utils/UrlUtils.java

```diff
@@ -0,0 +1,22 @@
+package io.flutter.utils;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+
+public class UrlUtils {
+    public static String addUrlTags(String input) {
```

Comment:
nit: add methods should add something to a list. generateHtmlFragmentWithHrefTags.